scripts/www/build-cfg-help.pl

```diff
@@ -323,7 +351,7 @@ ()
     } elsif (/^#/) {
         next;
     } elsif ($_ ne "") {
-        print "NOTICE: unknown line '$_'\n";
+        print "NOTICE: line $linecount: unknown line '$_'\n";
     }
 }
 end_options;
```

Comment:
Please report unclosed `IF`s here. We now have `@ifelse` we can trivially check...

Reply:
Done in[b01c6f5](https://github.com/squid-cache/squid/pull/1546/commits/b01c6f574b2fdc5ff5c722d88a506e2047bc703c)